docs/reference/elaborator-reflection.rst

```diff
@@ -74,17 +74,54 @@ We can generate identity functions at any concrete type using the same script:
 Interactively Building Elab Scripts
 ===================================
 
-To build an ``Elab`` script interactively, use the ``:elab`` command at the REPL.
-It takes the name of a hole as an argument.
-To list all available holes, use the command ``:m``.
-
-In interactive elaboration shell, the following commands are available:
--  ``:q`` - Quits the elaboration shell (gives up on proving current lemma).
--  ``:abandon`` - Same as :q
--  ``:state`` - Displays the current state of the term being constructed.
--  ``:term`` - Displays the current proof term complete with its yet-to-be-filled holes.
--  ``:undo`` - Undoes the last tactic.
--  ``:qed`` - Once the elaboration shell tells you "No more goals," you get to type this in celebration! (Completes the construction and exits the shell)
+You can build an ``Elab`` script interactively at the REPL.
+Use the command ``:metavars``, or ``:m`` for short, to list the available holes.
+Then, issue the ``:elab <hole>`` command at the REPL
+to enter the elaboration shell.
+
+At the shell, you can enter proof tactics to alter the proof state.
+It is recommended to issue ``:module Language.Reflection.Elab``
+prior to entering the elaboration shell in order to have the system-provided
+tactics available to you.
+When you have discharged all goals, you can complete the proof
+using the ``:qed`` command and receive in return an elaboration script
+that fills the hole.
+
+The interactive elaboration shell accepts a limited number of commands,
+including a subset of the commands understood by the normal Idris reply
+as well as some elaboration-specific commands.
+
+General-purpose commands that remain available in the elaboration shell:
+
+- ``:eval <EXPR>``, or ``:e <EXPR>`` for short, evaluates the provided expression
```

Comment:
`:e` isn't a command in the normal REPL, so I wouldn't say that it remains available, just that it is available.